packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -9,7 +9,7 @@ const moreOptionsLabel = 'Options';
 const selectors = {
 	// Block Inserter
 	// Note the partial class match. This is to support site and post editor. We can't use aria-label because of i18n. :(
-	blockInserterButton: `${ panel } button[class*="header-toolbar__inserter-toggle"]`,
+	blockInserterButton: `${ panel } button[class*="inserter-toggle"]`,
```

Comment:
This was broken recently due to a [change in Gutenberg](https://github.com/WordPress/gutenberg/commit/569241a3e1b7d0cc0567fccb92b1b57eb1b5b9c5#diff-0f64190407e5a202cfef71e90e6065e3c7f6ad4fcd8601e9f3f85df137b11a92R149). They updated the selector for the block inserter button for edit-site only. The selector was changed from `edit-site-header-toolbar__inserter-toggle` to `edit-site-header-edit-mode__inserter-toggle`. For edit-post, the selector is still `edit-post-header-toolbar__inserter-toggle`. 

The code changed here still needs to work correctly on both selectors. Given the differences, the text `inserter-toggle` is shared text. I was concerned that may be too generic, but it tests correctly for E2E test in both the edit-site and edit-post environments.

Reply:
This seems like a good change to me! I'm going to kick off a few more E2E test builds to just make sure there's not any unexpected failures there 👍